content/en/docs/topics/advanced.md

```diff
@@ -143,3 +143,68 @@ func main() {
 }
 
 ```
+
+## Storage backends
+
+By default, Helm stores release information in Secrets in the namespace of
+the release.
+
+### ConfigMap storage backend
+
+To enable the ConfigMap backend, you'll need to set the environmental variable
+`HELM_DRIVER` to `configmap`.
+
+You can set it in a shell as follows:
+
+```shell
+export HELM_DRIVER=configmap
+```
+
+If you want to switch from the default backend to the ConfigMap
+backend, you'll have to do the migration for this on your own. You can retrieve
+release information with the following command:
+
+```shell
+kubectl get secret --all-namespaces -l "owner=helm"
+```
+
+**PRODUCTION NOTES**: It is not recommended to use ConfigMaps for sensitive data as
+it is stored unencoded. Secrets data is stored base64-encoded.
+
+### SQL storage backend
+
+There is a beta SQL storage backend that stores release information in an SQL database.
+
+Using such a storage backend is particularly useful if your release information
+weighs more than 1MB (in which case, it can't be stored in ConfigMaps/Secrets
+because of internal limits in Kubernetes' underlying etcd key-value store).
+
+To enable the SQL backend, you'll need to deploy a SQL database and set the environmental
+variable `HELM_DRIVER` to `sql`. The DB details are set with the environmental variables
+`HELM_DRIVER_SQL_DIALECT` and `HELM_DRIVER_SQL_CONNECTION_STRING`.
+
+You can set it in a shell as follows:
+
+```shell
+export HELM_DRIVER=sql
+export HELM_DRIVER_SQL_DIALECT=postgres
+export HELM_DRIVER_SQL_CONNECTION_STRING=postgresql://helm-postgres:5432/helm?user=helm&password=changeme
+```
+
+> Note: Only Postgres is supported at this moment. This means that `HELM_DRIVER_SQL_DIALECT`
+needs to be set to `postgres`.
+
+**PRODUCTION NOTES**: It is recommended to:
```

Comment:
Do we want to tell them how to setup their database for production? Is there more general advice we should give them and then provide a link?